stubs/inertia/resources/js/Pages/Auth/TwoFactorChallenge.vue

```diff
@@ -82,6 +81,7 @@ const submit = () => {
                     class="mt-1 block w-full"
                     autocomplete="one-time-code"
                 />
+                <JetInputError class="mt-2" :message="form.errors.code" />
```

Comment:
It's a little unfortunate that this needs to be `code` rather than `recovery_code`, but it looks like Fortify puts the error here even when it's the recovery code that was attempted. The only time an error will appear under `recovery_code` is when the provided recovery code is not a string, and I don't think can ever happen in normal usage of the app.

Reply:
I've created a PR for Fortify to fix this: https://github.com/laravel/fortify/pull/401

If that PR is accepted, this will need to change to:

```suggestion
                <JetInputError class="mt-2" :message="form.errors.recovery_code" />
```


If that PR is not accepted, it might be a good idea to change this to something like this (untested):

```suggestion
                <JetInputError class="mt-2" :message="form.errors.recovery_code ?? form.errors.code" />
```